pick

```diff
@@ -50,17 +51,26 @@ class Pick():
       cmd= 'gpg --decrypt --batch --armor --passphrase %s %s' % (self.token, self.safe)
       p1 = subprocess.Popen(cmd.split(), stdout=subprocess.PIPE, stderr=null)
       secrets, err = p1.communicate()
-  
+
       if err:
         print err
         exit(102)
-  
+
     try:
       return json.loads(secrets)
     except ValueError:
       print 'Unable to read safe at %s' % self.safe
       exit(101)
 
+  def delete_password(self, alias):
+    ''' Deletes a password from the safe '''
+    alias = raw_input('Enter an Alias\n> ')
+    safe = self.read_safe()
+
```

Comment:
Right before you delete the password by alias, you'll want to ensure that alias exists of else you'll raise a `KeyError`.

The pattern I generally use is
```py
if not alias in safe['data']:
  print 'Alias does not exist'
  exit()
```